packages/core/src/login/webview/commonAuthViewProvider.ts

```diff
@@ -87,6 +87,8 @@ export class CommonAuthViewProvider implements WebviewViewProvider {
         // This will fire whenever the user opens or closes the login page from 'somewhere else'
         // i.e. NOT when switching from/to the chat window, which uses the same view area.
         webviewView.onDidChangeVisibility(async () => {
+            this.onDidChangeVisibility?.fire(webviewView.visible)
```

Comment:
there is another case on line 117, which means this would fire twice. is line 117 still needed?